test cases/vala/7 shared library/test.json

```diff
@@ -9,6 +9,8 @@
     {"type": "file", "file": "usr/include/installed_vala_onlyh.h"},
     {"type": "file", "file": "usr/share/vala/vapi/installed_vala_all.vapi"},
     {"type": "file", "file": "usr/share/vala-1.0/vapi/installed_vala_all_nolib.vapi"},
-    {"type": "file", "file": "usr/share/vala/vapi/installed_vala_onlyvapi.vapi"}
+    {"type": "file", "file": "usr/share/vala/vapi/installed_vala_onlyvapi.vapi"},
+    { "type": "python_file", "file": "usr/@PYTHON_PURELIB@/source.py" },
+    { "type": "python_bytecode", "file": "usr/@PYTHON_PURELIB@/source.py" }
```

Comment:
Wrong file?

Reply:
In the bytecode case it replaces `source.py` by `__pycache__/source.cpython-311.py`.